DQM/SiStripMonitorClient/scripts/LegendToQT.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python
+
+import Image, sys
+
+background = Image.open(sys.argv[1])
+foreground = Image.open(sys.argv[2])
+
+background.paste(foreground, (3000, 40), foreground)
```

Comment:
Hi @davidlange6 Sorry for the delay, the TKMAP is made by DQM/SiStripMonitorClient/test/SiStripDQM_OfflineTkMap_Template_cfg_DB.py. As @fioriNTU said, it is not easy to add root style legend in the script.